includes/class-wc-payments-order-service.php

```diff
@@ -17,11 +17,12 @@ class WC_Payments_Order_Service {
 	/**
 	 * Order status constants.
 	 */
-	const STATUS_CANCELLED = 'cancelled';
-	const STATUS_COMPLETED = 'completed';
-	const STATUS_FAILED    = 'failed';
-	const STATUS_ON_HOLD   = 'on-hold';
-	const STATUS_PENDING   = 'pending';
+	const STATUS_CANCELLED  = 'cancelled';
+	const STATUS_COMPLETED  = 'completed';
+	const STATUS_FAILED     = 'failed';
+	const STATUS_ON_HOLD    = 'on-hold';
+	const STATUS_PENDING    = 'pending';
+	const STATUS_PROCESSING = 'processing';
```

Comment:
Are those states correlating with WooCommerce core constants?

Reply:
yes, but since I won't use the constants anymore, I will undo the changes in this file.

Reply:
:+1:, mind creating a new low-priority issue (for a coll-down week, perhaps) to review and perhaps migrate the WooCommerce Payments codebase on WooCommerce core constants?

Reply:
Pre-condition for such migration is that constants from the core are including all of the values from this specific place.

Reply:
created an [issue](https://github.com/Automattic/woocommerce-payments/issues/5286) for this. Thank you!

Reply:
Thanks!